docs/modules/core/pages/material/normal-types.adoc

```diff
@@ -0,0 +1,56 @@
+# Normal Map Conventions
+
+There are two conventions for normal maps:
+
+* OpenGL (known as OpenGL-Style)
+* DirectX (known as DirectX-Style)
+
+The difference between OpenGL style and DirectX style is the Y value (green channel) is flipped, so OpenGL uses Y+ and DirectX is Y-.
+This can cause shading to not look quite right if a shader tries to use OpenGL convention on a normal map designed for DirectX.
+
+Luckily, JME supports both OpenGL and DirectX conventions by using a `NormalType` parameter in the Lighting and PBRLighting material
+definitions. The parameter can be set to either 1 (to use OpenGL) or -1 (to use DirectX).
+By default, JME uses the DirectX convention (NormalType is -1).
+
+### Using NormalType
+
+If you're using Lighting.j3md or PBRLighting.j3md for your materials, you can use the NormalType parameter to set
+which normal map convention your normal maps are using.
+
+```java
+Material mat = new Material(assetManager,
+        "Common/MatDefs/Light/Lighting.j3md");
+mat.setInt("NormalType", 1); //set the type to OpenGL
+```
+
+### Which Convention are My Normal Maps?
+
+If you know what software created your normal map, then you can tell which convention the map uses. Here are the default
+conventions of some common 3D programs:
+
+[cols="2"]
+|===
+<a| Software
+<a| Convention
+a| Blender
+a| OpenGL
+a| Maya
+a| OpenGL
+a| 3ds Max
+a| DirectX
+a| JMonkeyEngine
+a| DirectX
+a| Unity3D
+a| OpenGL
+a| Unreal Engine
+a| DirectX
+
+|===
+
```

Comment:
This is not always the case, those are the input normal type, what is outputted by blender, for example, depends on the addon used. 
Also, these programs are not commonly used to generate normalmaps as far as i know.
If we need to add a section with programs, we should probably show the setting window for substance painter or other similar software